feature-libs/cart/base/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -46,8 +46,11 @@ export class AddToCartComponent implements OnInit, OnDestroy {
   hasStock: boolean = false;
   inventoryThreshold: boolean = false;
 
-  showInventory$: Observable<boolean | undefined> | undefined =
-    this.component?.data$.pipe(map((data) => data.inventoryDisplay));
+  showInventory$:
+    | Observable<boolean | undefined>
+    | undefined = this.component?.data$.pipe(
+    map((data) => data.inventoryDisplay)
+  );
 
```

Comment:
Overall, did only the format change in this file?

Reply:
Right, only the formatting changed here